src/dmd/traits.d

```diff
@@ -139,10 +139,11 @@ shared static this()
         "getVirtualIndex",
         "getPointerBitmap",
         "isZeroInit",
-        "getTargetInfo"
+        "getTargetInfo",
+        "getLoc"
```

Comment:
Nit: I'd prefer `getLocation`.

Reply:
Chnaged

Reply:
This is still wrong.

Reply:
whoops, I changed the other one and not this one.